templates/admin/user/edit.tmpl

```diff
@@ -151,7 +151,7 @@
 
 				<div class="field">
 					<button class="ui green button">{{.locale.Tr "admin.users.update_profile"}}</button>
-					<div class="ui red button show-modal" data-modal="#delete-user-modal">{{.locale.Tr "admin.users.delete_account"}}</div>
+					<button type="button" class="ui red button show-modal" data-modal="#delete-user-modal">{{.locale.Tr "admin.users.delete_account"}}</button>
```

Comment:
The above "admin.auths.delete" button doesn't use the `type=button`, while other JS buttons need the `type=button`. It seems inconsistent.

The purpose of `type=button` is to prevent from submitting the form, could such "preventing" be done by JS? Then no need to write a lot of `type=button` in code.

Reply:
Oh yeah, looks like I saw this common pattern of `two buttons behind each other means accept and cancel button`, and thus added it here.
I can remove it again.

Reply:
Then the  JS must be fixed .....

Reply:
What JS?
At the moment, this is a browser native mechanism:
`<button>`s get an implicit `type="submit"` if none is set.
If I don't set `type="button"`, then it will submit the form as soon as the button is pressed, which otherwise only happens when I click on the `Delete` button inside the deletion dialogue, which is an implicit `submit` button as well.

Reply:
Fix `$('.show-modal').on('click', function () {`, add `e.preventDefault()`. Without the preventDefault, indeed there is a JS bug.